// my-component-library/esm5/my-component-library.js
import { Component, NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

var HeaderComponent = (function () {
    function HeaderComponent() {
    }
    HeaderComponent.prototype.ngOnInit = function () {
    };
    return HeaderComponent;
}());
HeaderComponent.decorators = [
    { type: Component, args: [{
                selector: 'app-header',
                template: "<p>\n  header works!\n</p>\n",
                styles: [""]
            },] },
];
HeaderComponent.ctorParameters = function () { return []; };
var HeaderModule = (function () {
    function HeaderModule() {
    }
    return HeaderModule;
}());
HeaderModule.decorators = [
    { type: NgModule, args: [{
                imports: [
                    CommonModule
                ],
                declarations: [HeaderComponent],
                exports: [HeaderComponent]
            },] },
];
HeaderModule.ctorParameters = function () { return []; };

export { HeaderModule, HeaderComponent as ɵa };
